engine/getstarted/last_page.md

```diff
@@ -20,6 +20,10 @@ Depending on your interest, the Docker documentation contains a wealth of inform
     <th class="tg-031e">Where to find it</th>
   </tr>
   <tr>
+    <td class="tg-031e">Tutorials and sample app walkthroughs</td>
+    <td class="tg-031e"><a href="/engine/getstarted-voting-app/">Define and deploy a multi-container app</a>. A nice next step after this Getting Started tutorial, this is a sample app walkthrough that shows you how to use a docker stack file to define a set of services and deploy them to a swarm.</td>
```

Comment:
Maybe there is a way to avoid the clause "A nice....tutorial,":

<link>Define and deploy a multi-container-app</link> is a logical next step after finishing the Getting Started Tutorial. It walks you through using a Docker stack file to define a set of services and deploy them on a swarm.